src/style.css

```diff
@@ -0,0 +1,84 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.container__box {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
+  min-width: 300px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: white;
+  
+}
+
+.container__box:nth-child(1) {
+  background-color: rgb(100%, 0%, 0%);
+  order: 1;
+}
+
+.container__box:nth-child(2) {
+  background-color: rgb(80%, 0%, 0%);
+  order: 2;
+}
+
+.container__box:nth-child(3) {
+  background-color: rgb(60%, 0%, 0%);
+  order: 3;
+}
+
+.container__box:nth-child(4) {
+  background-color: rgb(40%, 0%, 0%);
+  order: 4;
+}
+
+.container__box:nth-child(5) {
+  background-color: rgb(20%, 0%, 0%);
+  order: 5;
+}
+
+.container__box:nth-child(6) {
+  background-color: rgb(0%, 0%, 0%);
+  order: 6;
+}
+
+@media (min-width: 0) {
+  .container__box {
+    flex-basis: 100%;
+  }
+}
```

Comment:
If you want to use a mobile-first approach, style your blocks with 100% width by default. Don't add too many media queries